logstash-core/logstash-core.gemspec

```
@@ -57,7 +57,7 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency "mustermann", '~> 1.0.3'
   gem.add_runtime_dependency "sinatra", '~> 2.1.0' # pinned until GH-13777 is resolved
   gem.add_runtime_dependency 'puma', '~> 5'
-  gem.add_runtime_dependency "jruby-openssl", "~> 0.11.0"
+  gem.add_runtime_dependency "jruby-openssl", "~> 0.11"
```

are we allowing 0.11.x on concerns about 0.12.x? otherwise we can do instead:

```suggestion
  gem.add_runtime_dependency "jruby-openssl", "~> 0.12", ">= 0.12.1"
```